### PR TITLE
Fix the CouchDB shell scripts

### PR DESCRIPTION
**Pull request by swansontec**
- Use the correct script location in the example config file.
- Use heredocs to avoid weird indentation on the target machines.
- Block access from the public internet.
- Add a config file for Couch itself to write its settings to.

### CODE REVIEW

File: couch/install.ubuntu.sh [hunk 1]
```diff
@@ -33,12 +33,24 @@ sudo systemctl stop couchdb
...
 
+# Block access from the public internet:
+sudo cat > /opt/couchdb/etc/local.d/10-bind-address.ini <<EOF
+# Only allow access through Caddy
+[chttpd]
+bind_address = 127.0.0.1
+EOF
+
+# Create a file for the Couch admin interface to write to:
+cat > /opt/couchdb/etc/local.d/99-auto.ini <<EOF
```
Comment (samholmes) [inline couch/install.ubuntu.sh:44]: I couldn't find any documentation on this file from my internet searches. What is `99-auto.ini`?

File: couch/install.ubuntu.sh [hunk 2]
```diff
@@ -33,12 +33,24 @@ sudo systemctl stop couchdb
...
+# Couch itself will write to this file
+EOF
+
 # Add seedlist config if present
 if [ -n $COUCH_SEEDLIST ]; then
-  sudo echo "
+  sudo cat > /opt/couchdb/etc/local.d/10-clustering.ini <<EOF
```
Comment (samholmes) [inline couch/install.ubuntu.sh:50]: Why is 10 added to the name, `10-clustering.ini`? Just curious.

**Review by samholmes — APPROVED** [review]

File: couch/install.ubuntu.sh (hunk 2 above)
Comment (swansontec) [reply couch/install.ubuntu.sh:50]: It's customary to put numbers in front of the files in a config.d directory, to control the sort order.

File: couch/install.ubuntu.sh (hunk 1 above)
Comment (swansontec) [reply couch/install.ubuntu.sh:44]: 99 is the highest number, so this config file comes "last". Therefore, this is where Couch itself will write settings.

**Pull request merged**